components/HorizontalScroller.js

```diff
@@ -1,68 +1,52 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { ChevronLeft } from '@styled-icons/fa-solid/ChevronLeft';
-import { ChevronRight } from '@styled-icons/fa-solid/ChevronRight';
-import themeGet from '@styled-system/theme-get';
+import { ArrowBack } from '@styled-icons/material/ArrowBack';
+import { ArrowForward } from '@styled-icons/material/ArrowForward';
 import styled from 'styled-components';
 
 import { debounceScroll } from '../lib/ui-utils';
 import withViewport from '../lib/withViewport';
 
-/** Main chevrons container, spacing the items properly */
-const ChevronsContainer = styled(({ onPrevClick, onNextClick, ...props }) => (
-  <div {...props}>
-    <ChevronLeft onMouseDown={onPrevClick} disabled={!onPrevClick} />
-    <ChevronRight onMouseDown={onNextClick} disabled={!onNextClick} />
-  </div>
-))`
-  display: flex;
-  justify-content: space-between;
-  color: #dadada;
-  user-select: none;
-  width: ${props => props.size * 3.5}px;
+import { Box, Flex } from './Grid';
 
-  svg {
-    width: ${props => props.size}px;
-    height: ${props => props.size}px;
+const RefContainer = styled(Box)`
+  display: flex;
+  overflow-x: auto;
+  scroll-behavior: smooth;
+  padding: 20px;
```

Comment:
Adding this padding here creates some visual diffs. For example, it adds a scrollbar on https://staging.opencollective.com/babel with 1920x1080 resolution. Unless it's serving a specific purpose, can we get rid of it?

Reply:
Yep removed 👍🏻